pkgs/by-name/py/pyditz/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  python3Packages,
+  fetchPypi,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "pyditz";
+  version = "0.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-2gNlrpBk4wxKJ1JvsNeoAv2lyGUc2mmQ0Xvn7eiaJVE=";
+  };
+
+  nativeBuildInputs = with python3Packages; [ setuptools-scm ];
+
+  propagatedBuildInputs = with python3Packages; [
+    pyyaml
+    six
+    jinja2
+    cerberus
+  ];
+
```

Comment:
Could you please check if we can run the tests? Perhaps it's required to switch from PyPI to the upstream source.

-> https://hg.sr.ht/~zondo/pyditz/browse/tests?rev=tip